server/backend/users/views.py

```
@@ -130,6 +127,68 @@
+
+    def post(self, request, uidb64, token):
+        password = request.data.get("password")
+        if not password:
+            return Response({"error": "Password is required"}, status=status.HTTP_400_BAD_REQUEST)
+
+        try:
+            uid = urlsafe_base64_decode(uidb64).decode()
+            user = User.objects.get(pk=uid)
+
+            if not PasswordResetTokenGenerator().check_token(user, token):
+            if not PasswordResetTokenGenerator().check_token(user, token):
```

Ruff (0.8.2)

SyntaxError: Expected an indented block after `if` statement

---

Trailing comma missing

(COM812)